src/views/App/scrollConfig.tsx

```diff
@@ -0,0 +1,6 @@
+const scrollBlackList = ['system-profile'];
```

Comment:
Nitpick: this file can be `.ts`, not `.tsx`.

Reply:
Ah yep you're right :)

Reply:
Renamed in https://github.com/CMSgov/easi-app/pull/1437/commits/7ef92399064e1f01b41071aff2398f6b6b4d0b12.